Utilities/StrFmt.cpp

```diff
@@ -214,6 +214,25 @@ void fmt_class_string<std::vector<char8_t>>::format(std::string& out, u64 arg)
 	out.append(obj.cbegin(), obj.cend());
 }
 
+template <>
+void fmt_class_string<fmt::buf_to_hexstring>::format(std::string& out, u64 arg)
+{
+	const auto& _arg = get_object(arg);
+	out.reserve(out.size() + (_arg.len * 3));
+	static constexpr char hex[16] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'};
+
+	for (usz index = 0; index < _arg.len; index++)
+	{
+		out += hex[_arg.buf[index] >> 4];
```

Comment:
Load it once from the memory please, this is for use of vm memory in case other threads write to it - so it won't print values which were never present in memory and cause confusion.